geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -9990,6 +10008,40 @@ public EntryExpiryTask getEntryExpiryTask(Object key) {
     return br.getEntryExpiryTask(key);
   }
 
+  PartitionRegionConfig updatePRNodeInformation() {
+    PartitionRegionConfig newConfig = null;
+    Region<String, PartitionRegionConfig> partitionedRegionRoot = getPRRoot();
```

Comment:
refactor this guy to share code with the gateway methods. It would be nice if we just had one method on this class that does the locking, fetches config, calls updateprconfig, unlocks